2022-08-22 07:25:58,166 INFO    Thread-22 (_run_job):25949 [wandb_setup.py:_flush():76] Configure stats pid to 25949
2022-08-22 07:25:58,166 INFO    Thread-22 (_run_job):25949 [wandb_setup.py:_flush():76] Loading settings from /home/codespace/.config/wandb/settings
2022-08-22 07:25:58,166 INFO    Thread-22 (_run_job):25949 [wandb_setup.py:_flush():76] Loading settings from wandb/settings
2022-08-22 07:25:58,166 INFO    Thread-22 (_run_job):25949 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'entity': 'raihaan123', 'project': 'PINNs', 'root_dir': '/workspaces/UROP-Notes', 'run_id': 'an9bix6k', 'sweep_param_path': '/workspaces/UROP-Notes/wandb/sweep-53ssclns/config-an9bix6k.yaml', 'sweep_id': '53ssclns'}
2022-08-22 07:25:58,166 INFO    Thread-22 (_run_job):25949 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'src/runner.py', 'program': '/workspaces/UROP-Notes/src/runner.py'}
2022-08-22 07:25:58,166 INFO    Thread-22 (_run_job):25949 [wandb_init.py:_log_setup():461] Logging user logs to /workspaces/UROP-Notes/wandb/run-20220822_072558-an9bix6k/logs/debug.log
2022-08-22 07:25:58,166 INFO    Thread-22 (_run_job):25949 [wandb_init.py:_log_setup():462] Logging internal logs to /workspaces/UROP-Notes/wandb/run-20220822_072558-an9bix6k/logs/debug-internal.log
2022-08-22 07:25:58,167 INFO    Thread-22 (_run_job):25949 [wandb_init.py:init():495] calling init triggers
2022-08-22 07:25:58,167 INFO    Thread-22 (_run_job):25949 [wandb_init.py:init():498] wandb.init called with sweep_config: {'depth': 22, 'height': 78, 'num_domain': 2900}
config: {}
2022-08-22 07:25:58,167 INFO    Thread-22 (_run_job):25949 [wandb_init.py:init():548] starting backend
2022-08-22 07:25:58,167 INFO    Thread-22 (_run_job):25949 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-22 07:25:58,172 INFO    Thread-22 (_run_job):25949 [backend.py:ensure_launched():217] starting backend process...
2022-08-22 07:25:58,172 INFO    Thread-22 (_run_job):25949 [backend.py:ensure_launched():222] started backend process with pid: 28970
2022-08-22 07:25:58,173 INFO    Thread-22 (_run_job):25949 [wandb_init.py:init():558] backend started and connected
2022-08-22 07:25:58,179 INFO    Thread-22 (_run_job):25949 [wandb_run.py:_config_callback():1131] config_cb None None {'depth': 22, 'height': 78, 'num_domain': 2900}
2022-08-22 07:25:58,181 INFO    Thread-22 (_run_job):25949 [wandb_init.py:init():634] updated telemetry
2022-08-22 07:25:58,187 INFO    Thread-22 (_run_job):25949 [wandb_init.py:init():665] communicating run to backend with 30 second timeout
2022-08-22 07:25:59,202 INFO    Thread-22 (_run_job):25949 [wandb_run.py:_on_init():1919] communicating current version
2022-08-22 07:25:59,232 INFO    Thread-22 (_run_job):25949 [wandb_run.py:_on_init():1923] got version response upgrade_message: "wandb version 0.13.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-08-22 07:25:59,233 INFO    Thread-22 (_run_job):25949 [wandb_init.py:init():700] starting run threads in backend
2022-08-22 07:25:59,298 INFO    Thread-22 (_run_job):25949 [wandb_run.py:_console_start():1893] atexit reg
2022-08-22 07:25:59,299 INFO    Thread-22 (_run_job):25949 [wandb_run.py:_redirect():1766] redirect: SettingsConsole.REDIRECT
2022-08-22 07:25:59,299 INFO    Thread-22 (_run_job):25949 [wandb_run.py:_redirect():1771] Redirecting console.
2022-08-22 07:25:59,299 ERROR   Thread-22 (_run_job):25949 [wandb_run.py:_redirect():1830] Failed to redirect.
Traceback (most recent call last):
  File "/opt/python/3.10.4/lib/python3.10/site-packages/wandb/sdk/wandb_run.py", line 1823, in _redirect
    out_redir.install()
  File "/opt/python/3.10.4/lib/python3.10/site-packages/wandb/sdk/lib/redirect.py", line 702, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/opt/python/3.10.4/lib/python3.10/site-packages/wandb/sdk/lib/redirect.py", line 652, in add_fd
    self._register()
  File "/opt/python/3.10.4/lib/python3.10/site-packages/wandb/sdk/lib/redirect.py", line 637, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/opt/python/3.10.4/lib/python3.10/signal.py", line 56, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread of the main interpreter
2022-08-22 07:25:59,300 INFO    Thread-22 (_run_job):25949 [wandb_init.py:init():732] run started, returning control to user process
